src/views/CreateProfile.vue

```diff
@@ -10,359 +10,290 @@
+              <a
+                href="https://addons.mozilla.org/en-US/firefox/addon/superhero-wallet/"
+                target="_blank"
+              >
+                here
+              </a><br>
+              Click
+              <span class="highlited">Add</span>
+              to Firefox. Confirm by clicking
+              <span class="highlited">Add</span> again in the message prompt.<br>
+              Then the Superhero icon should appear on the upper right
+              corner of your Firefox browser
+              tool bar.<br>
+            </p>
+            <p>
+              For Chrome, Brave, and Opera:<br>
+              Get the extension
+              <a
+                :href="chromeLink"
+                target="_blank"
+              >
+                here
+              </a><br>
+              Add the extension to your browser.<br>
+              Then the Superhero icon should appear on the
+              upper right corner of your Chrome browser tool bar.
+            </p>
+          </template>
+        </expandable-block>
+        <expandable-block>
+          <template slot="title">
+            CREATING YOUR WALLET
+          </template>
+          <template slot="body">
+            <p>
+              Tick the checkbox to agree to the terms and conditions.<br>
+              Click <span class="highlited">CREATE A NEW WALLET.</span><br>
+              That’s it—your wallet is created!<br>
+              You can now start receiving tips from anyone through your wallet address.<br>
+            </p>
+            <p>
+              You can find your wallet address on your dashboard, right above your balance.<br>
+              It’s a random string of numbers and letters that starts with ak_.<br>
+              This wallet address is also referred to as a public
+              key— it’s what others see as your public ID by default.<br>
+              You can easily copy this by clicking the copy
+              button right next to your wallet address.
+            </p>
+          </template>
+        </expandable-block>
+        <expandable-block>
+          <template slot="title">
+            SECURING YOUR WALLET
+          </template>
+          <template slot="body">
+            <p>
+              With great freedom comes great responsibility.<br>
+              You’re in charge of keeping the keys to your account.<br>
+              There are no third-party custodians to store
+              nor connect any password with any account.<br>
+              The task of keeping the keys to your wallet is yours and yours alone, Superhero.
+            </p>
+            <p>
+              Here’s how to secure and backup your wallet:<br>
+              Once you’re in your dashboard, you’ll get a series
+              of words called a seed phrase.<br>
+              The seed phrase is somewhat like a password –
+              it’s a unique set of words (in order) which connects you to your wallet.<br>
+              <ul>
```

Comment:
semantically `<ul/ol>'s are not supposed to be inside `<p>` though browsers do not usually complain